screens/Menu.js

```diff
@@ -9,15 +10,21 @@ export default class Menu extends React.Component {
   }
 
   changeLocationSettings = async () => {
-    //Opens up settings so user can toggle their location on or off
-      const supported = await Linking.canOpenURL("app-settings:")
-        if (!supported) {
-            alert("Cannot open app settings - please open settings manually to toggle location.");
-          } else {
-            Linking.openURL("app-settings:");
-          }
+    if (Platform.OS === "ios") {
+      //Opens up settings so user can toggle their location on or off
+      const supported = await Linking.canOpenURL("app-settings:");
+      if (!supported) {
+        alert(
+          "Cannot open app settings - please open settings manually to toggle location."
+        );
+      } else {
+        Linking.openURL("app-settings:");
+      }
+    } else {
+      const intent = await IntentLauncher.startActivityAsync(IntentLauncher.ACTION_MANAGE_APPLICATIONS_SETTINGS);
```

Comment:
is this the android line? is there any case where it wouldn't be supported like with ios above?